astro/src/content/json/applications/search-response.json

```diff
@@ -170,5 +209,6 @@
       }
     }
   ],
+  "expandable": [],
```

Comment:
```suggestion
  "expandable": ["roles", "scopes"],
```

Reply:
This response contains `scopes` and `roles`, so those do not appear in the `expandable` list. It only contains expansion options that were not used.

Reply:
Ah, thanks. I didn't understand this field.